src/main/utilities/headers/ModelImport.hpp

```diff
@@ -23,20 +25,23 @@ using std::ifstream;
 */
 class ModelImport {
  public:
-      explicit ModelImport(string, vector<string>, vector<GLint>, GLuint);
-      Polygon *createPolygonFromFile();
-      int processLine(string, int, Polygon *, bool);
-      Polygon *buildObject(int objectId, bool oglConfigure);
-      void configureOpenGl(Polygon *, int);
+      explicit ModelImport(string, vector<string>, vector<GLint>, GLuint, GfxController *);
+      Polygon createPolygonFromFile();
+      inline Polygon getPolygon() { return polygon_; }
+      int processLine(string, int);
+      int buildObject(int objectId);
+      void configureOpenGl(Polygon &, int);
```

Comment:
At some point in the future we want to remove the `configureOpenGl` method. This name is a misnomer, because it actually makes GfxController calls, not OpenGL calls. This functionality should be moved to the scene object level. The SceneObjects (GameObject, TextObject, ColliderObject) should keep track of their own VAOs and VBOs. Storing these in the Polygon class unnecessarily inflates it.